.github/workflows/deploy.yml

```diff
@@ -0,0 +1,27 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
```

Comment:
Should we install NodeJS in a proper version or this is done under the hood?

(I don't have experience - this is why I'm asking)

```
    - uses: actions/setup-node@v2
      with:
        node-version: 16
```

Reply:
Yeah, you're right. I forgot about that.
Without it, it would use the default version installed on [ubuntu-latest](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md) (which would be 16.14.0)